src/App.jsx

```diff
@@ -14,45 +16,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const ALPHABET_SORT_FIELD = 'alphabet';
+const LENGTH_SORT_FIELD = 'length';
+const DEFAULT_SORT_FIELD_VALUE = '';
+
+function getPreparedGoods(goods, { sortField, isReversed }) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((firstGood, secondGood) => {
+      switch (sortField) {
+        case ALPHABET_SORT_FIELD:
+          return firstGood.localeCompare(secondGood);
+
+        case LENGTH_SORT_FIELD:
+          return firstGood.length - secondGood.length;
+
+        default:
+          return 0;
```

Comment:
In that case, the sort method will do nothing. Consider moving `switch...case` outside of the sort callback to avoid extra calls.